### PR TITLE
Adding info about combining css classes

### PR DESCRIPTION
**Pull request by rom-1**
Adding info about combining css classes using comma separation

Closes #

Changes proposed in this pull request:

-
-
-

How to test the feature manually:

1.
2.
3.

Pull request checklist:

- [ ] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

**Review by aledeg — APPROVED** [review]
LGTM.

File: docs/en/users/05_Configuration.md [hunk 1]
```diff
@@ -182,9 +182,25 @@ Here we find that the block that encompasses nothing but the content of the arti
...
+<p class='add'>unwanted content</p>
+<p class='content'>wanted content</p>
+</div>
+```
+In this case it is possible to combine multiple CSS tags using comma separation: ```#article p.content, #article h2```
+
 ### Retrieve a truncated stream with external tools
 
-Complimentary tools can be used to retrieve full article content, such as:
+Complimentary tools can be used to retrieve full article content, such as:
```
Comment (aledeg) [inline docs/en/users/05_Configuration.md]: Could you just remove the extra space at the end?

Comment (rom-1) [reply docs/en/users/05_Configuration.md]: done 👍

File: docs/en/users/05_Configuration.md [hunk 2]
```diff
@@ -182,6 +182,22 @@ Here we find that the block that encompasses nothing but the content of the arti
 *  Lesnumériques: ```article#body div.text.clearfix```
 *  Phoronix: ```#main .content```
 
+#### Combining CSS Classes
+Lets say we have an article which contains ads within its content, and we do not want to have those ads retrieved by FreshRSS. Example Source Code:
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]: Let's is a contraction of let us.
```suggestion
Let's say we have an article which contains ads within its content, and we do not want to have those ads retrieved by FreshRSS. Example Source Code:
```

File: docs/en/users/05_Configuration.md [hunk 3]
```diff
@@ -182,6 +182,22 @@ Here we find that the block that encompasses nothing but the content of the arti
...
+```
+<div id='article'>
+<h2>wanted</h2>
+<p class='content'>wanted content</p>
+<p class='add'>unwanted content</p>
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]:
```suggestion
<p class='ad'>unwanted content</p>
```

File: docs/en/users/05_Configuration.md [hunk 4]
```diff
@@ -182,6 +182,22 @@ Here we find that the block that encompasses nothing but the content of the arti
...
+<h2>wanted</h2>
+<p class='content'>wanted content</p>
+<h2>wanted</h2>
+<p class='add'>unwanted content</p>
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]:
```suggestion
<p class='ad'>unwanted content</p>
```

File: docs/en/users/05_Configuration.md [hunk 5]
```diff
@@ -182,6 +182,22 @@ Here we find that the block that encompasses nothing but the content of the arti
...
+<p class='content'>wanted content</p>
+</div>
+```
+In this case it is possible to combine multiple CSS tags using comma separation: ```#article p.content, #article h2```
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]:
```suggestion
In this case it's possible to combine multiple CSS selectors with a comma: ```#article p.content, #article h2```
```

**Review by Frenzie — CHANGES_REQUESTED** [review]

**Review by Frenzie — APPROVED** [review]

**Pull request merged**